docker/Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM ubuntu:20.04 as web-base
+
+RUN apt update && apt install -y git curl
```

Comment:
Should use `apt-get` instead of plain `apt` to get rid of the warning "WARNING: apt does not have a stable CLI interface. Use with caution in scripts."

Reply:
sure!